frontend/lib/swr-cache/index.ts

```diff
@@ -212,7 +212,7 @@ export const withCache = <
          return res.status(200).json({ success: true });
       } catch (error) {
          logger.error(printError(error));
```

Comment:
It's possible that we no longer want the logging in the line above because likely whatever is above us in the callstack will do that. Does it appear to double-log it with the current code?